test/tests/functional/pbs_release_limited_res_suspend.py

```diff
@@ -660,3 +660,72 @@ def test_node_custom_res_release_upon_suspension(self):
 
         self.assertEqual(rv[0][ras_bar], "40",
                          msg="pbs should not release resource bar")
+
+    def test_resuming_with_no_res_released(self):
+        """
+        Set restrict_res_to_release_on_suspend to a resource that a job
+        does not request and then suspend this running job using qsig
+        check if such a job resumes when qsig -s resume is issued
+        """
+        # Set foo in restrict_res_to_release_on_suspend server attribute
+        a = {ATTR_restrict_res_to_release_on_suspend: 'mem'}
+        self.server.manager(MGR_CMD_SET, SERVER, a)
+
+        j1 = Job(TEST_USER)
+        j1.set_attributes({ATTR_l+'.ncpus': '4'})
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+
+        # suspend job
+        self.server.sigjob(jobid=jid1, signal="suspend")
+
+        job = self.server.status(JOB, id=jid1)
+        self.assertNotEqual(job, None)
```

Comment:
No need to check for None.  status() will raise a PtlStatusError exception if there is a problem.  You'll never get here if status() returns None.